ansible_collections/arista/avd/requirements-dev.txt

```
@@ -8,7 +8,7 @@ PyYAML>=6.0.0
 # temporarily capping pydantic for CI
 pydantic<2.12
 # dev requirements
-ansible-core>=2.16.0,<2.19.0
+ansible-core>=2.16.0,<2.20.0
```

ansible-core 2.19 in another PR as well.

```suggestion
ansible-core>=2.16.0,<2.19.0
```